es/elastic/api.py

```diff
@@ -120,9 +120,13 @@ def get_valid_table_view_names(self, type_filter: str) -> "Cursor":
             for item in response:
                 # First column is TABLE_NAME
                 if item["index"] == self._get_value_for_col_name(result, "name"):
-                    if int(item["docs.count"]) == 0:
-                        is_empty = True
-                        break
+                    try:
+                        if int(item["docs.count"]) == 0:
+                            is_empty = True
+                            break
+                    except Exception as ex:
```

Comment:
too broad exception, replace it by `TypeError`. Also it seems less clear what can be expected from the ES index.

what do you think about:
``` python
if item["docs.count"] is None or item["docs.count"] in (0, "0")
```